app/routes/index.js

```diff
@@ -1,127 +1,123 @@
-	app.use('/games/search', access.isAuthenticated, require('./games/search'));
-	app.use('/groups/add', access.isAdmin, require('./groups/add'));
-	app.use('/games', access.isAuthenticated, require('./games/index'));
-	app.use('/releases', access.isEditor, require('./releases/release'));
-	app.use('/archive', access.isEditor, require('./games/index'));
-	app.use('/groups/group', access.isAuthenticated, require('./groups/group'));
-	app.use('/groups/search', access.isAdmin, require('./groups/search'));
-	app.use('/groups', access.isAdmin, require('./groups/index'));
-	app.use('/users/search', require('./users/search'));
-	app.use('/users/add', access.isAdmin, require('./users/add'));
-	app.use('/users', access.isAdmin, require('./users/index'));
-	app.use('/configuration', access.isAdmin, require('./configuration'));
+  // Site pages
+  app.use('/', require('./home'));
+  app.use('/embed', require('./embed'));
+  app.use('/docs', access.isAuthenticated, require('./docs'));
+  app.use('/games/add', access.isEditor, require('./games/add'));
+  app.use('/games/search', access.isAuthenticated, require('./games/search'));
+  app.use('/groups/add', access.isAdmin, require('./groups/add'));
+  app.use('/games', access.isAuthenticated, require('./games/index'));
+  app.use('/releases', access.isEditor, require('./releases/release'));
+  app.use('/archive', access.isEditor, require('./games/index'));
+  app.use('/groups/group', access.isAuthenticated, require('./groups/group'));
+  app.use('/groups/search', access.isAdmin, require('./groups/search'));
+  app.use('/groups', access.isAdmin, require('./groups/index'));
+  app.use('/users/search', require('./users/search'));
+  app.use('/users/add', access.isAdmin, require('./users/add'));
+  app.use('/users', access.isAdmin, require('./users/index'));
+  app.use('/configuration', access.isAdmin, require('./configuration'));
 
-	// RESTful service for releases
-	app.use('/api/release', require('./api/release'));
-	app.use('/api/releases', require('./api/releases'));
-	app.use('/api/games', require('./api/games'));
-	// Authentication Pages
-	app.use('/login', access.isAnonymous, require('./login'));
-	app.use('/logout', access.isAuthenticated, require('./logout'));
-	app.use('/register', access.isAnonymous, require('./register'));
-	app.use('/forgot', access.isAnonymous, require('./forgot'));
-	app.use('/reset', access.isAnonymous, require('./reset'));
-	app.use('/profile', access.isAuthenticated, require('./profile'));
-	app.use('/password', access.isAuthenticated, require('./password'));
+  // RESTful service for releases
+  app.use('/api/release', require('./api/release'));
+  app.use('/api/releases', require('./api/releases'));
+  app.use('/api/games', require('./api/games'));
+  // Authentication Pages
+  app.use('/login', access.isAnonymous, require('./login'));
+  app.use('/logout', access.isAuthenticated, require('./logout'));
+  // app.use('/register', access.isAnonymous, require('./register'));
```

Comment:
@chipbell4 Is this a route we want to hold onto at all, or should we have this be deleted instead of commented out?

Reply:
Yeah, let's remove that.